rubicon_ml/client/mixin.py

```diff
@@ -259,13 +261,20 @@ def artifact(self, name=None, id=None):
                 )
 
             artifact = artifacts[-1]
+            return artifact
```

Comment:
same comment as with the experiment getting function - this first part needs to be inside the try too to handle getting by name and ID